src/enact/gradio/gradio.py

```diff
@@ -340,12 +340,12 @@ def __init__(
         input required exception.
     """
```

Comment:
nit: Above `Type[Any]` could just be `Type`

Reply:
Done